packages/formStructure/components/FormSectionHeader.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import { SpacingBox } from "../../styleUtils/modifiers";
+import { HeadingText2, Text } from "../../styleUtils/typography";
+
+interface FormSectionHeaderProps {
+  title: string;
+  subtitle?: React.ReactNode;
```

Comment:
Since it's inside a `<Text>` element already, what do you think about making this `React.ReactNode | string`?

Reply:
`React.ReactNode` includes `string`